src/modules/launcher/Plugins/Microsoft.Plugin.Registry/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "303417D927BF4C97BCFFC78A123BE0C8",
+  "ActionKeyword": "*",
```

Comment:
Is "*" an action keyword or is it just a placeholder?

Reply:
It's gives all queries to the plugin. Without any filtering.

Reply:
I have to try use `ActionKeywords: [ "HK", "hk", "Hk", "hK" ]`, but WOX always print the keyword + search in the ComboBox.
![2020-11-27 09_24_35-PowerLauncher](https://user-images.githubusercontent.com/17874713/100427397-aa923880-3092-11eb-82d2-8227f6807a5c.png)
It is not possible to override this behavior. Currently I always override the `result.QueryTextDisplay` to catch short base keys.